pkg/arvo/app/link-view.hoon

```diff
@@ -205,19 +205,37 @@
   ^-  (list card)
   ?.  ?=(%accepted -.upd)  ~
   ?.  =(/link path.upd)    ~
-  :~  ::  sync the group
+  =/  =group-id
+    (need (group-id:de-path:group-store path.invite.upd))
+  :~
+      ::  add self
+      :*  %pass
+          [%group-add path.invite.upd]
+          %agent  [ship.group-id %group-hook]
+          %poke  %group-action
+          !>  ^-  action:group-store
+          [%add-members group-id (sy our.bowl ~) ~]
```

Comment:
The `%add-members` here feels a little bit weird. Have you considered having a special action for "I am joining this group"?

Reply:
What would that give us over and above what we already get from `%add-members`?

Reply:
Cleaner separation of the flows/use cases for the action. Right now there's a bunch of implicit logic that comes with "add members", where you have to go "admins can do this, but also randos can do this, but only if they add themselves, and only if the policy allows it".